src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsSourceTest.java

```diff
@@ -40,4 +43,32 @@ public void testGetReadPCollectionLocal(){
         DataflowAssert.thatSingleton(count).isEqualTo(7L);
         p.run();
     }
+
+    @Test
+    public void testGetInvalidPCollectionLocal() {
+        // ValidationStringency.SILENT should prevent any read error even though the input has what looks like invalid reads.
+        Pipeline p = TestPipeline.create();
+        ReadsSource source = new ReadsSource(hiSeqBam, p);
+        SAMFileHeader header = source.getHeader();
+        final SAMSequenceDictionary sequenceDictionary = header.getSequenceDictionary();
+        DataflowWorkarounds.registerGenomicsCoders(p);
+        PCollection<Read> reads = source.getReadPCollection(IntervalUtils.getAllIntervalsForReference(sequenceDictionary), ValidationStringency.SILENT);
+        PCollection<Long> count = reads.apply(Count.globally());
+        DataflowAssert.thatSingleton(count).isEqualTo(1674L);
+        p.run();
+    }
+
+    @Test(expectedExceptions = RuntimeException.class)
```

Comment:
Could we narrow this to the actual thrown exception?  I know dataflow will wrap it into a runtime exception, but testing for `RuntimeException` is not terribly useful since breaking the test almost always ends up producing a different runtime exception.  I think it would be better to catch the exeption within the test and unwrap it.